Makefile

```diff
@@ -78,22 +81,32 @@ frontend_quality:
 
 
 
-tests: backend_tests
+tests: backend_tests ## Run all tests
 
-backend_tests:
+backend_tests: ## Run python tests
 	@echo "🍜 Running python tests"
 	${DOCKER_COMPOSE_TEST} up -d neo4j
 	${DOCKER_COMPOSE_TEST}  run --rm taxonomy_api pytest /parser /parser
 	${DOCKER_COMPOSE_TEST}  run --rm taxonomy_api pytest /code/tests
 	${DOCKER_COMPOSE_TEST} stop neo4j
 
-checks: quality tests
+checks: quality tests ## Run all checks (quality + tests)
 
 
 #------------#
 # production #
 #------------#
 
-create_external_volumes:
+create_external_volumes: ## Create external volumes (production only)
 	@echo "🍜 Creating external volumes (production only) …"
 	docker volume create ${COMPOSE_PROJECT_NAME}_neo4j-data
+
+
+#------------#
+# utils #
+#------------#
+
+.PHONY: build up dev lint backend_lint frontend_lint quality backend_quality frontend_quality tests backend_tests checks create_external_volumes help
+
+help: ## Description of the available commands
+	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
```

Comment:
Cool, I propose we also add a `.DEFAULT_GOAL := help`

Also comment this line.

```suggestion
# this command will grep targets and the following ## comment will be used as description
help: ## Description of the available commands
	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
	
.DEFAULT_GOAL := help
```